astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -29,6 +29,10 @@
 from .baseradec import BaseRADecFrame
 from .cirs import CIRS
 from .ecliptic import *  # there are a lot of these so we don't list them all explicitly
+from .ecliptic import (
+    BarycentricMeanEcliptic, BarycentricTrueEcliptic, BaseEclipticFrame, CustomBarycentricEcliptic,
+    GeocentricMeanEcliptic, GeocentricTrueEcliptic, HeliocentricEclipticIAU76,
+    HeliocentricMeanEcliptic, HeliocentricTrueEcliptic)
```

Comment:
The `from .ecliptic import *` line serves no purpose because the contents of `__all__` are imported explicitly anyways: 
https://github.com/astropy/astropy/blob/63aefe6955db7d82208638ce60f47266a1be270d/astropy/coordinates/builtin_frames/ecliptic.py#L11-L15

Reply:
✅